clients/src/test/java/org/apache/kafka/clients/admin/MockAdminClient.java

```diff
@@ -1014,7 +1014,9 @@ synchronized public DescribeLogDirsResult describeLogDirs(Collection<Integer> br
                 for (Node node : nodes) {
                     Map<String, LogDirDescription> logDirDescriptionMap = unwrappedResults.get(node.id());
                     LogDirDescription logDirDescription = logDirDescriptionMap.getOrDefault(partitionLogDirs.get(0), new LogDirDescription(null, new HashMap<>()));
-                    logDirDescription.replicaInfos().put(new TopicPartition(topicName, topicPartitionInfo.partition()), new ReplicaInfo(0, 0, false));
+                    Map<TopicPartition, ReplicaInfo> topicPartitionReplicaInfoMap = new HashMap<>(logDirDescription.replicaInfos());
+                    topicPartitionReplicaInfoMap.put(new TopicPartition(topicName, topicPartitionInfo.partition()), new ReplicaInfo(0, 0, false));
+                    logDirDescriptionMap.put(partitionLogDirs.get(0), new LogDirDescription(logDirDescription.error(), topicPartitionReplicaInfoMap));
```

Comment:
Instead of setting unknown bytes for `totalBytes` and `usableBytes` for the new `LogDirDescription` object, I think we should use the `totalBytes` and `usableBytes` as well from the existing `LogDirDescription` object.

Reply:
Good point, thanks for the feedback. I've made this change.

Reply:
Thanks